cmd/server.go

```diff
@@ -120,7 +120,7 @@ var Serve = &cobra.Command{
 		eventHandlerConfig := events.Config{
 			UpstreamConf: upstreamConfig,
 		}
-		go events.ListenForEvents(context.Background(), eventHandlerConfig)
+		go events.ListenForEvents(context.Background(), db.Gorm, eventHandlerConfig)
```

Comment:
We're passing db object down to 5 functions, do you think we should create an event handler struct ?

```
eventHandler = EventHandler{db: db.Gorm, ctx: context, config: config}

go eventHandler.ListenForEvents()
````

Reply:
Sure, I'd prefer that actually.

Reply:
great 👍🏽